res/controllers/Roland_DJ-202-scripts.js

```diff
@@ -252,8 +252,51 @@ DJ202.Deck = function (deckNumbers, offset) {
 
     
     // ============================= TRANSPORT ==================================
-    this.play = new components.PlayButton([0x90 + offset, 0x00]); // LED doesn't stay on
     this.cue = new components.CueButton([0x90 + offset, 0x01]);
+    this.play = new components.Button({
```

Comment:
Nice! Though  I'd personally expect it to start again if I hit play during braking. But that's probably a matter of preference again.

Reply:
I also triggered this by accident once or twice, and it might be more of a gimmick. Maybe we’ll add a config setting to disable/enable this at some point.